### DISCUSSION TITLE
Education showing 1969

### DISCUSSION THREAD

**Discussion by FrazzApps** [microsoft-search/pnp-modern-search#3064] [Q&A]
On the popup when  hovering over a people result the education show shows that it is from 1969-current.  If nothing is set in the profile this should be blank.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/4583163/dbbf043e-b966-4299-9590-8bc0db67ed52)

**Comment by kasperbolarsen** [comment]
Please check if this is the result you get from the API.  Is that is the case then it is not related to this project.

**Comment by wobba** [comment]
This seems to be a Graph Toolkit issue for that component, not a web part issue.
